src/config.ts

```diff
@@ -8,4 +8,5 @@ export const config: IPerfumeConfig = {
   maxTime: 30000,
   // web-vitals report options
   reportOptions: {},
+  enableNavigationBasedActiveSteps: false,
```

Comment:
@Cdub-9 I guess the docs for `enableNavigationBasedActiveSteps` will come in the next PR? if that so, that's ok.

Reply:
Btw, `enableNavigationBasedActiveSteps` is such a odd name. What else can we play out with?

Reply:
I am thinking more about this, wonder if we should actually have this `true` as default, then name the variable like `disableNavigationTracking` or something like that

Reply:
went with `enableNavigationTracking` instead, let me know what you think